Cargo.toml

```diff
@@ -12,7 +12,7 @@ resolver = "2"
 
 [workspace.package]
 edition = "2021"
-version = "0.2.0"
+version = "0.3.0"
```

Comment:
**question:** Should we bump this to 0.4 so that it matches the lib version? Should we always keep these two the same?

I believe there's been previous discussions on this topic before, is that correct?

Reply:
I don't recall a previous discussions on this topic. I am okay with either ways. 

>  Should we always keep these two the same?
Maybe not as crates and binaries can evolve independently imo.

Reply:
I think there was a discussion on keeping the release of each shim, as well as the lib, independent, as their versions could diverge.